service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/docmosis/TranslationRequestFormCreationServiceDocmosisTest.java

```diff
@@ -0,0 +1,231 @@
+        generatedContentOutputFile = "TranslationFormRequestLayoutIsOther.";
+
+        request = DocmosisTranslationRequest.builder()
+            .layout(DocmosisWelshLayout.builder()
+                .other(true)
+                .build())
+            .build();
+
+        DocmosisDocument docmosisDocumentPDF = underTest.buildTranslationRequestDocuments(request.toBuilder()
+            .format(RenderFormat.PDF)
+            .build()
+        );
+
+        generateDocuments(docmosisDocumentPDF);
+
+        assertThat(remove(extractPdfContent(docmosisDocumentPDF.getBytes())))
+            .isEqualToNormalizingWhitespace(getExpectedText(expectedContentFileLocation));
+    }
+
+    private DocmosisTranslationRequest buildTranslationRequest(boolean isEnglishToWelsh,
+                                                               String filepathToCountWords, LocalDate dateOfReturn) {
+        return DocmosisTranslationRequest.builder()
+            .familyManCaseNumber("FamilyManCaseNumber123")
+            .name("Family Public Law System")
+            .description("Test description.".repeat(2))
+            .contactInformation("contactfpl@justice.gov.uk")
+            .department("Family Public Law")
+            .project(DocmosisWelshProject.builder()
+                .reform(true)
+                .build())
+            .layout(DocmosisWelshLayout.builder()
+                .mirrorImage(true)
+                .build())
+            .translate(DocmosisTranslateLanguages.builder()
+                .englishToWelsh(isEnglishToWelsh)
+                .welshToEnglish(!isEnglishToWelsh)
+                .build())
+            .wordCount(countWordsInFile(filepathToCountWords))
+            .dateOfReturn(dateOfReturn)
+            .build();
+    }
+
+    private void generateDocuments(
+        DocmosisDocument docmosisDocumentPDF) throws IOException {
+
+        storeToOuputFolder(
+            generatedContentOutputFile.concat(RenderFormat.PDF.getExtension()),
+            docmosisDocumentPDF.getBytes()
+        );
+
+        storeToOuputFolder(
+            generatedContentOutputFile.concat(RenderFormat.WORD.getExtension()),
+            underTest.buildTranslationRequestDocuments(request.toBuilder()
+                .format(RenderFormat.WORD)
+                .build()
+            ).getBytes()
+        );
+    }
+
+    private int countWordsInFile(String testFileLocation) {
+        String line;
+        int numberOfWords = 0;
+
+        try {
+            FileReader file = new FileReader(testFileLocation);
+            BufferedReader bufferedReader = new BufferedReader(file);
+
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] words = line.split(" ");
+                numberOfWords = numberOfWords + words.length;
+            }
+            bufferedReader.close();
+        } catch (IOException exception) {
+           return 0;
+        }
+
+        return numberOfWords;
+    }
```

Comment:
The countWordsInFile has not been finalised. I have to wait until the ticket has been updated to specify what exactly it is counting.